src/hbb/xsecs.py

```diff
@@ -0,0 +1,379 @@
+xsecs["TTto2L2Nu"] = 923.6 * 0.333 * 0.333  # = 102.41 (96.9) - 91.29
+xsecs["TTtoLNu2Q"] = 923.6 * 2 * (0.667 * 0.333)  # = 410.28 (404.0) - 405.1
+
+# Diboson
+xsecs["WW"] = 116.8  #  173.4 (116.8 at NNLO)
+xsecs["WZ"] = 54.3
+xsecs["ZZ"] = 16.7
+xsecs["ZZto2L2Q_TuneCP5_13p6TeV_powheg-pythia8"] = 2.36
+xsecs["ZZto2Nu2Q_TuneCP5_13p6TeV_powheg-pythia8"] = 4.48
+xsecs["ZZto4L_TuneCP5_13p6TeV_powheg-pythia8"] = 0.170
+xsecs["ZZto2L2Nu_TuneCP5_13p6TeV_powheg-pythia8"] = 0.674
+xsecs["WZtoLNu2Q_TuneCP5_13p6TeV_powheg-pythia8"] = 12.368
+xsecs["WZto2L2Q_TuneCP5_13p6TeV_powheg-pythia8"] = 3.696
+xsecs["WZto3LNu_TuneCP5_13p6TeV_powheg-pythia8"] = 1.786
+xsecs["WWto4Q_TuneCP5_13p6TeV_powheg-pythia8"] = 78.79
+xsecs["WWtoLNu2Q_TuneCP5_13p6TeV_powheg-pythia8"] = 76.16
+xsecs["WWto2L2Nu_TuneCP5_13p6TeV_powheg-pythia8"] = 18.41
+# xsecs["WWto4Q_1Jets-4FS"]
+# xsecs["WZto4Q-1Jets-4FS"]
+
+# SingleTop
+xsecs["TbarBQ_t-channel_4FS"] = 87.2
+xsecs["TBbarQ_t-channel_4FS"] = 145.0
+xsecs["TbarWplusto4Q"] = 10.8
+xsecs["TWminusto4Q"] = 10.8
+xsecs["TbarWplustoLNu2Q"] = 10.7
+xsecs["TWminustoLNu2Q"] = 10.7
+xsecs["TbarWplusto2L2Nu"] = 2.62
+xsecs["TWminusto2L2Nu"] = 2.62
+
+# Higgs
+xsecs["GluGluHto2B_M-125"] = 51.53 * BR_HBB  # 29.93
+# SX: took XSDB NLO number (0.5246) and multiplied it by the NNLO/NLO ratio for inclusive ggH from 13 TeV
+xsecs["GluGluHto2B_PT-200_M-125"] = 0.5246 * (43.92 / 27.8) * BR_HBB
+# https://twiki.cern.ch/twiki/bin/view/LHCPhysics/LHCHWG136TeVxsec_extrap
+xsecs["VBFHto2B_M-125_dipoleRecoilOn"] = 4.023 * BR_HBB  # 2.34
+xsecs["WminusH_Hto2B_Wto2Q_M-125"] = 0.8889 * BR_WQQ * BR_HBB  # 0.349
+xsecs["WminusH_Hto2B_WtoLNu_M-125"] = 0.8889 * BR_WLNU * BR_HBB  # 0.167
+xsecs["WplusH_Hto2B_Wto2Q_M-125"] = 0.5677 * BR_WQQ * BR_HBB  # 0.222
+xsecs["WplusH_Hto2B_WtoLNu_M-125"] = 0.5677 * BR_WLNU * BR_HBB  # 0.106
+xsecs["ZH_Hto2B_Zto2L_M-125"] = 0.8079 * BR_ZLL * BR_HBB
+xsecs["ZH_Hto2B_Zto2Q_M-125"] = 0.8079 * BR_ZQQ * BR_HBB
+xsecs["ggZH_Hto2B_Zto2L_M-125"] = 0.1360 * BR_ZLL * BR_HBB
+xsecs["ggZH_Hto2B_Zto2Nu_M-125"] = 0.1360 * BR_ZLNU * BR_HBB
+xsecs["ggZH_Hto2B_Zto2Q_M-125"] = 0.1360 * BR_ZQQ * BR_HBB
+xsecs["ttHto2B_M-125"] = 0.5700 * BR_HBB
+
+# Triple-higgs
+# ggHHHto6B_13TeV         0.0894e-3 (not including BR_HBB)
+xsecs["HHHTo6B_c3_0_d4_0"] = 3.707e-05 * BR_HBB * BR_HBB * BR_HBB
+xsecs["HHHTo6B_c3_0_d4_99"] = 0.005855 * BR_HBB * BR_HBB * BR_HBB
+xsecs["HHHTo6B_c3_0_d4_minus1"] = 4.117e-05 * BR_HBB * BR_HBB * BR_HBB
+xsecs["HHHTo6B_c3_19_d4_19"] = 0.1476 * BR_HBB * BR_HBB * BR_HBB
+# SM sample
+xsecs["HHHTo6B_c3_1_d4_0"] = 2.908e-05 * BR_HBB * BR_HBB * BR_HBB
+xsecs["HHHTo6B_c3_1_d4_2"] = 1.616e-05 * BR_HBB * BR_HBB * BR_HBB
+# xsecs["HHHTo6B_c3_2_d4_minus1"] =
+# xsecs["HHHTo6B_c3_4_d4_9"] =
+xsecs["HHHTo6B_c3_minus1_d4_0"] = 0.0001127 * BR_HBB * BR_HBB * BR_HBB
+xsecs["HHHTo6B_c3_minus1_d4_minus1"] = 0.0001087 * BR_HBB * BR_HBB * BR_HBB
+xsecs["HHHTo6B_c3_minus1p5_d4_minus0p5"] = 0.0001941 * BR_HBB * BR_HBB * BR_HBB
+
+# Di-Higgs
```

Comment:
remove all the di-higgs and triple higgs cross sections, keep only the cross sections of the samples we have been using